modules/api/src/main/java/org/apache/fluo/api/config/SimpleConfiguration.java

```diff
@@ -192,34 +195,30 @@ public void load(InputStream in) {
    * @since 1.2.0
    */
   public void load(File file) {
-    try {
-      PropertiesConfiguration config = new PropertiesConfiguration();
-      // disabled to prevent accumulo classpath value from being shortened
-      config.setDelimiterParsingDisabled(true);
-      config.load(file);
+    try(InputStream in = Files.newInputStream(file.toPath())) {
+      PropertiesConfiguration config = newPropertiesConfiguration();
+      new FileHandler(config).load(cleanUp(in));
```

Comment:
Again, no need to use FileHandler... you should use PropertyConfigurationLayout:

```java
config.getLayout().load(config, cleanUp(in));
```

Reply:
Done. When I actually when to use this I noticed I had to repeat the config and found that a bit odd.